pkg/contracts/src/CVStrategy.sol

```diff
@@ -951,7 +950,7 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
 
     function setRegistryCommunity(address _registryCommunity) external onlyPoolManager(msg.sender) {
         registryCommunity = RegistryCommunity(_registryCommunity);
-        //@todo missing event, also we can change it?
```

Comment:
great

Reply:
the last question is: "allow chaging registry community could potentialy present risks and vuln? the change guarantee that CVStrategy will keep working?"

Reply:
No not for now, I wanted to talk to you about this Im not sure if its a good idea, we will need to transfer all memberPowerData for a strategy to the new registry and stuff, lots of things come with a change I think. should we just remove ?

Reply:
yeah can be removed